### DISCUSSION TITLE
Could not load type 'Swashbuckle.AspNetCore.Swagger.Info' from assembly 'Swashbuckle.AspNetCore.Swagger'

### DISCUSSION THREAD

**Discussion by sibinthomask** [domaindrivendev/Swashbuckle.AspNetCore#3199] [Show and tell]
I am trying to migrate the Core WebApi project from .Net core 2.1 to .Net 8.0. As we do usually, I tried to add packages from NuGet. To solve errors, I installed Swashbuckle,Swashbuckle.AspNetCore,Swashbuckle.AspNetCore.SwaggerGen,Swashbuckle.AspNetCore.SwaggerUI,Swashbuckle.AspNetCore.Filters.
Now I am getting this error - 
TypeLoadException: Could not load type 'Swashbuckle.AspNetCore.Swagger.Info' from assembly 'Swashbuckle.AspNetCore.Swagger, Version=7.2.0.0, Culture=neutral, PublicKeyToken=62657d7474907593'.
Please share your ideas to solve this error

**Comment by martincostello** [comment]
Duplicate of #3198.

**Reply by sibinthomask** [reply]
Sir, I am looking for an answer. That's why again I put this question.

**Reply by martincostello** [reply]
An exception message is not a question. You were asked to provide a minimal reproducible example: https://github.com/domaindrivendev/Swashbuckle.AspNetCore/issues/3198#issuecomment-2537976668
